### PR TITLE
Modify to improve code readability

### PR DESCRIPTION
**Pull request by ping035627**
This PR modify codes to improve code readability.

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Please rebase

**Comment by ping035627** [comment]
I have rebased, and everything up-to-date.

**Comment by AkihiroSuda** [comment]
image

@ping035627 Please rebase again

**Comment by ping035627** [comment]
sorry, I just operate error, and I have rebased again, thanks.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: cli/command/image/build.go [hunk 1]
```diff
@@ -138,18 +138,14 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 	var (
 		buildCtx io.ReadCloser
 		err      error
```
Comment (thaJeztah) [inline cli/command/image/build.go]: These should align now with the other lines;

```go
	var (
		buildCtx      io.ReadCloser
		err           error
		contextDir    string
		tempDir       string
		relDockerfile string
		progBuff      io.Writer
		buildBuff     io.Writer
	)
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Comment by ping035627** [comment]
@thaJeztah ok, I have modified, thanks for reminding.

**Comment by tiborvass** [comment]
LGTM

**Review by tiborvass — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]

**Pull request merged**